tests/setup.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+from setuptools import setup
+from os import path
+
+PACKAGE_NAME = "ccftools"
+
+package_dir = path.join(path.abspath(path.dirname(__file__)), PACKAGE_NAME)
+print(package_dir)
+with open(path.join(package_dir, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version="0.11.10",
+    description="Set of tools and utilities for the Confidential Consortium Framework (CCF)",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/CCF",
+    license="Apache License 2.0",
+    author="CCF Team",
+    author_email="ccfeng@microsoft.com",
```

Comment:
Is that set up?

Reply:
I have not setup an account on PyPi with this email address yet. As I understand it, this email address can be different from the one of the account we actually use to upload to PyPi. This will be addressed in a future PR once we automate the release.